test-support/helpers/sl/utils/utils.js

```diff
@@ -53,27 +53,27 @@ var convertStringToArray = function( underTest ) {
  *
  * Property names are only extracted from the object provided.  No recursion into nested objects occurs.
  *
- * @function convertObjectKeysToArray
- * @param   {object} underTest
- * @throws  {Ember.assert} If argument is not provided or is not an object
- * @returns {array}
+ * @function
+ * @param {Object} underTest
+ * @throws {Ember.assert} If argument is not provided or is not an object
+ * @returns {Array}
  */
-var convertObjectKeysToArray = function( underTest ) {
+let convertObjectKeysToArray = function( underTest ) {
 
-    Ember.assert( 'Object must be supplied', 'object' === typeof underTest && !Array.isArray( underTest ) );
+    Ember.assert( 'Object must be supplied', 'object' === Ember.typeOf( underTest ) && !Array.isArray( underTest ) );
```

Comment:
there are more object types than just object that typeOf returns now that using it